pypak/__init__.py

```
@@ -152,24 +152,24 @@ def __super_import__(module_named_x):
 				if __sys_path__ not in sys.path:
 					sys.path.insert(0, __sys_path__)
 	except Exception:
-		raise ImportError("pak failed to import.")
+		raise ImportError("pypak failed to import.")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)